lib/resty/mlcache/ipc.lua

```diff
@@ -13,6 +13,7 @@ local find         = string.find
 local min          = math.min
 local type         = type
 local pcall        = pcall
+local error        = error
```

Comment:
Ha, those are only for cold paths so I omitted the caching of `error`, but why not since `mlcache` does it as well